course/models.py

```diff
@@ -365,20 +365,17 @@ class ParticipationTag(models.Model):
     course = models.ForeignKey(Course,
             verbose_name=_("Course"), on_delete=models.CASCADE)
     name = models.CharField(max_length=100,
+            blank=False, null=False,
             # Translators: name format of ParticipationTag
-            help_text=_("Format is lower-case-with-hyphens. "
-            "Do not use spaces."),
+            help_text=_("Should be a valid identifier."),
             verbose_name=_("Name of participation tag"))
     shown_to_participant = models.BooleanField(default=False,
-            verbose_name=_("Shown to pariticpant"))
+            verbose_name=_("Shown to participant"))
 
     def clean(self):
         super(ParticipationTag, self).clean()
 
-        import re
-        name_valid_re = re.compile(NAME_VALID_REGEX)
```

Comment:
Any other uses of `NAME_VALID_REGEX` can probably also b removed in favor of `.isidentifier()`.

Reply:
I agree.

Reply:
Done.